### PR TITLE
Link to docs for configuring an iop deployment

### PR DESCRIPTION
**Pull request by michielbdejong**
Linking to information we were missing while working on #38.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ Kubernetes Charts Repository for CS3ORG
 
 This reposity contains the official charts for CS3Org software, like Reva.
 
+See also [configuring an iop deployment](https://developer.sciencemesh.io/docs/technical-documentation/iop/deployment/kubernetes/#configuring-an-iop-deployment).
```
Comment (SamuAlfageme) [inline README.md:7]: Can you move this line towards the example at the end? I think it makes semantically much more sense to include there, next to the revad installation command.

**Review by SamuAlfageme — CHANGES_REQUESTED** [review]
Thanks a bunch for your contribution @michielbdejong! Indeed, it's a great point to include that link on the README.

File: README.md (hunk 1 above)
Comment (michielbdejong) [reply README.md]: Yes! Done.

**Comment by SamuAlfageme** [comment]
@michielbdejong CI is failing since it thinks it's a chart update, if you can include `[skip ci]` on the commit message like in https://github.com/cs3org/charts/commit/23425adfff483fe5252f564189cac695da45dd3e - thanks!

**Pull request merged**